lmod_proxy/edx_grades/__init__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+"""Blue print for handling the ``POST`` from the edx-platform
+REMOTE_GRADEBOOK feature.
+"""
+from flask import (
+    Blueprint,
+)
+
+edx_grades = Blueprint(
+    'lmod_proxy_edx_grades',
+    __name__,
+    template_folder='templates/edx_grades',
+    static_folder='static/edx_grades',
+)
+
+
+@edx_grades.route('/', methods=['GET'])
```

Comment:
yeah, when I write code to handle the post, I'll add that method, planning on using WTForm to validate the edx post and on the get provide a user interface to exercise the form similar to how it is now, but spruced up a bit.